src/components/Tabs/index.js

```diff
@@ -0,0 +1 @@
+export { default } from './Tabs';
```

Comment:
Here it looks like the only thing this component exposes to a developer is `Tabs`.
But in fact we also provide `Tab`, `TabMenu` and `TabItem`. I'd specify it here to make it more clear.

And furthermore, now (as far as we expose 4 components) it looks a bit confusing. 
To make it more clear I'd suggest the following structure (split TabsMenu and Tabs to two different components, so that TabsMenu is a "atom" and  Tabs is a "molecule"):

```
TabsMenu
├── TabItem
├── TabMenu.js
├── __tests__
└── index.js

Tabs
├── Tab
├── TabContent
├── Tabs.js
├── __tests__
└── index.js
```

Reply:
I'm not really looking forward to reorganizing this structure (I'm short on time). I think it makes sense that `TabMenu` is housed under `Tabs`.. And everything you need is exported and listed in `src/index.js` anyway, right?

Reply:
File structure is indeed not a small change, but I think at least we should export everything that supposed to be exposed

Reply:
But... everything that needs to be exposed is exported? `src/index.js`